scripts/publish_docs.sh

```diff
@@ -0,0 +1,54 @@
+make
+if [[ $APIS_EMAIL != "" ]]
+then
+  git config --global user.email "${APIS_EMAIL}"
+  git config --global user.name "${APIS_USERNAME}"
+  tag_name="${GITHUB_REF##*/}"
+  LOCAL=0
+else
+  tag_name="local_test"
+  LOCAL=1
+fi
+
+rm -rf generated_docs docs
+mkdir generated_docs
+
+if [[ $LOCAL == 1 ]]
+then
+  dstrm --generate-docs
+else
+  ./dist/dstrm --generate-docs
+fi
+
+cd generated_docs
+find . -name "*strm_*" -exec sh -c 'mv "$1" "${1#*strm_}"' _ {} \;
+rm -rf dstrm.md
+for i in $(find . -name '*.md'); do
+  DIRNAME_TO_PARSE="${i%_*}"
+  FILENAME="${i##*_}"
+  if [[ $DIRNAME_TO_PARSE != *md ]]
+  then
+    DIRNAME=$(echo "${DIRNAME_TO_PARSE%%.md}" | sed -r 's/_/\//g')
+    mkdir -p "$DIRNAME"
+    mv "$i" "$DIRNAME/$FILENAME"
+  else
+    mkdir -p "${FILENAME%.md}"
+    mv "$FILENAME" "${FILENAME%.md}/index.md"
+    rm -rf "${DIRNAME_TO_PARSE}"
+  fi
+done
+
+cd ..
+
+if [[ $GITHUB_TOKEN == "" ]]
```

Comment:
I'd make this last part optional, on whether or not you're running this script locally.

Reply:
For testing what the output cli docs look like, it is necessary to publish to a branch. I'll will change it to that behaviour